docs/configuration.asciidoc

```diff
@@ -387,6 +387,32 @@ This table outlines the attributes and their corresponding permissions:
 
 |===
 
+[float]
+==== Configuring this library's logging policy
+
+By default, this library's internal logs are all printed for a debuggable app build.
+In the case of non-debuggable builds, only logs starting from level INFO onwards are printed.
+
+In case you would like to have a more customized policy, or even disable all of the logs from this library altogether, you could do so by providing your own `LoggingPolicy` config like so:
```

Comment:
In general, we try to avoid using gerunds (words ending in `ing`) in section/page headings. "Configure" is also in the page heading, so I don't think we need it here. Can we simplify this to "Logging policy"? Or is that removing too much context?

```suggestion
==== Logging policy

By default, all logs created by this library are printed for a debuggable app build.
In the case of non-debuggable builds, only logs at the INFO level and above are printed.

If you would like to create a custom log policy or even disable all of the logs from this library altogether, you can do so by providing your own `LoggingPolicy` configuration. The below example policy will allow all logs of level WARN and higher to be printed. Levels below WARN will be ignored.
```

Reply:
Thanks, it makes sense! - For the header, I think we should at least add the word `Internal` at the beginning to emphasize that this is for internal logs only and it has nothing to do with [OTel logs](https://opentelemetry.io/docs/concepts/signals/logs/). Apart from that, I think it's looking great 👍